controllers/admin/AdminCustomerThreadsController.php

```diff
@@ -1054,7 +1054,7 @@ public function syncImap()
                     if (Validate::isLoadedObject($ct) && ((isset($matches2[1]) && $ct->token == $matches2[1]) || $new_ct)) {
                         $message = imap_fetchbody($mbox, $overview->msgno, 1);
                         $message = quoted_printable_decode($message);
-                        $message = utf8_encode($message);
+                        $message = iconv(mb_detect_encoding($message),'UTF-8',$message);
```

Comment:
Hi,
Thank you for your PR. 

Please wether use Tools::iconv and please also test function_exists('mb_detect_encoding') and leave back to default utf8_encode if not. 

Best regards